Description: Fixes path to rootwrap.d
 The default config file was using /etc/quantum, this fixes it so that it uses
 /etc/neutron.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-08-07

--- neutron-2013.2~b2.orig/etc/rootwrap.conf
+++ neutron-2013.2~b2/etc/rootwrap.conf
@@ -4,7 +4,7 @@
 [DEFAULT]
 # List of directories to load filter definitions from (separated by ',').
 # These directories MUST all be only writeable by root !
-filters_path=/etc/quantum/rootwrap.d,/usr/share/quantum/rootwrap
+filters_path=/etc/neutron/rootwrap.d
 
 # List of directories to search executables in, in case filters do not
 # explicitely specify a full path (separated by ',')
